frontend/src/components/Issue/rollback/common.ts

```diff
@@ -58,12 +58,20 @@ export const useRollbackLogic = () => {
     }
     const database = databaseOfTask(task.value);
     const { engine, engineVersion } = database.instance;
-    if (engine !== "MYSQL") {
-      return "NONE";
-    }
-    if (!semverCompare(engineVersion, MIN_ROLLBACK_SQL_MYSQL_VERSION, "gte")) {
-      return "NONE";
+    switch (engine) {
+      case "MYSQL":
+        if (
+          !semverCompare(engineVersion, MIN_ROLLBACK_SQL_MYSQL_VERSION, "gte")
+        ) {
+          return "NONE";
+        }
+        break;
+      case "ORACLE":
```

Comment:
plz comment why we did nothing here when "ORACLE"

Reply:
Done.